torch/csrc/jit/serialization/unpickler.cpp

```diff
@@ -868,6 +868,21 @@ void Unpickler::rebuildTensor(bool quantized) {
     impl->set_storage_offset(storage_offset);
     impl->set_sizes_and_strides(size, stride);
     result = autograd::make_variable(result, requires_grad);
+
+    // Handle if math_bits were pickled.
+    // See `args` of _reduce_ex_internal
+    // for a regular tensor (final else case).
+    // Tensors pickled before this patch didn't
+    // have this argument for storing MathBits,
+    // in that case, we do nothing.
+    // NOTE: `math_bits` is the 7th arg.
+    // NOTE: This is only meant for regular tensor and not quantized!
+    if (!quantized && elements.size() == 7) {
+      // We skip the backward_hooks pointed at idx + 1.
+      auto math_bits = elements.at(idx + 2).toGenericDict();
```

Comment:
It would be more idiomatic to idx++ to skip the backward hooks, similar to how things are handled above